Xamarin.Forms.Platform.Android/Renderers/WebViewRenderer.cs

```diff
@@ -15,10 +15,13 @@ namespace Xamarin.Forms.Platform.Android
 {
 	public class WebViewRenderer : ViewRenderer<WebView, AWebView>, IWebViewDelegate
 	{
-		bool _ignoreSourceChanges;
+		public const string AssetBaseUrl = "file:///android_asset/";
+
+		FormsWebViewClient _webViewClient;
 		FormsWebChromeClient _webChromeClient;
 
-		IWebViewController ElementController => Element;
+		public IWebViewController ElementController => Element;
+		public bool IgnoreSourceChanges { get; set; }
```

Comment:
protected?

Reply:
I made these public so they can be accessed by the FormsWebViewClient. Could be protected internal instead.